zetaclient/chains/evm/observer/v2_inbound.go

```diff
@@ -129,16 +130,16 @@ func (ob *Observer) parseAndValidateDepositEvents(
 	// collect and sort events by block number, then tx index, then log index (ascending)
 	events := make([]*gatewayevm.GatewayEVMDeposited, 0)
```

Comment:
consider calling this validEvents

Reply:
renamed `events -> validEvents` https://github.com/zeta-chain/node/pull/3517/commits/38a30a86bceca54199f255b64b4737ff515ae3e4